llvm/lib/Transforms/Instrumentation/MemorySanitizer.cpp

```diff
@@ -528,6 +530,26 @@ static const PlatformMemoryMapParams NetBSD_X86_MemoryMapParams = {
     &NetBSD_X86_64_MemoryMapParams,
 };
 
+// SPIR Linux
+static const MemoryMapParams Intel_SPIR_MemoryMapParams = {
+    0, // AndMask
+    0, // XorMask (not used)
+    0, // ShadowBase (not used)
+    0, // OriginBase
+};
+
+static const PlatformMemoryMapParams Intel_GFX_MemoryMapParams = {
```

Comment:
variable with "Intel_" prefix is not a good idea, maybe we can rename it to 'SPIR_GEN_'?

Reply:
I think variable with "Intel_" prefix is fine, since there're some variables whose prefix is "AMD" in AddressSanitizer pass.